Source/Session.swift

```diff
@@ -984,7 +989,6 @@ extension Session: RequestDelegate {
         retrier.retry(request, for: self, dueTo: error) { retryResult in
             self.rootQueue.async {
                 guard let retryResultError = retryResult.error else { completion(retryResult); return }
-
```

Comment:
Please restore this newline between the guard and the `let retryError` line.

Reply:
Restored.